benchmark/webshop/evaluate_webshop.py

```diff
@@ -88,7 +88,7 @@ def get_runned_ids(file_path):
     args = parser.parse_args()
     rewards = []
     all_task_ids = list(range(0, 251))
-    REWARD_LOG_FILE = f"{args.llm}_{args.agent_arch}_results_webshop.csv"
+    REWARD_LOG_FILE = f"{args.llm.replace('/', '_')}_{args.agent_arch}_results_webshop.csv"
```

Comment:
This is needed because Hugging Face model repos are in the form `{org}/{repo_name}` which causes problems when trying to write the file to disk.